spring/spring/pom.xml

```diff
@@ -237,6 +237,43 @@
                     </instructions>
                 </configuration>
             </plugin>
+            <plugin>
```

Comment:
Clarified... Spring 4 works a bit differently because of Spring commons package. Everything is fine.